packages/components/src/components/ConfirmOnDevice/ConfirmOnDevice.tsx

```diff
@@ -212,7 +220,11 @@ export const ConfirmOnDevice = ({
             <Right>
                 <CloseWrapper>
                     {onCancel && (
-                        <Close onClick={onCancel} data-test="@confirm-on-device/close-button">
+                        <Close
+                            elevation={nextElevation[elevation]}
```

Comment:
Not sure about this, I think nextElevation shall not be used normally. Increase in elevation hall be done in Provider exclusively.

If you need it for some "hack" please explain the hack in the comment.